### ISSUE TITLE
POST `{{HOST}}/skill-search/users` is not working

### ISSUE THREAD

**Issue by codejamtc** [topcoder-platform/u-bahn-api#120]
![image](https://user-images.githubusercontent.com/42398485/132229200-9a8fbe80-c538-4984-a272-c47a22e4001f.png)

**Comment by sachin-maheshwari** [comment]
@phead198708 / @yoution getting following error - 
![Uploading Screenshot 2021-09-06 at 7.31.12 PM.png…]()

**Comment by sachin-maheshwari** [comment]
@yoution regarding this ?

**Comment by yoution** [comment]
@sachin-maheshwari @codejamtc I didnot change the code about this controller and service, the request body in the postman is outdated, I think you should add `organizationId` in the payload

**Comment by sachin-maheshwari** [comment]
I agree with @yoution, same issue are coming on Prod, so we can ignore.  <img width="986" alt="Screenshot 2021-09-08 at 10 56 28 AM" src="https://user-images.githubusercontent.com/6467888/132457577-4250c679-e905-4d5f-ac49-4d02486ef3fb.png">
